# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/experimental/TypedData.h
# Commit: 180daaed9cc8
# Full Hash: 180daaed9cc8c4691427f7b34361f9ce078944e3
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Fill in the rest of the ArrayBuffer+ArrayBufferView hierarchy, and base more ArrayBufferViewObject public APIs off of new API r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119857
# ==============================================================================

diff -r 410e13fdf4e5 -r 180daaed9cc8 js/public/experimental/TypedData.h
--- a/js/public/experimental/TypedData.h	Thu Aug 12 20:44:15 2021 +0000
+++ b/js/public/experimental/TypedData.h	Thu Aug 12 20:44:16 2021 +0000
@@ -387,6 +387,7 @@
     }
     return ArrayBuffer(nullptr);
   }
+  static ArrayBuffer unwrap(JSObject* maybeWrapped);
 
   static ArrayBuffer create(JSContext* cx, size_t nbytes);
 